src/west/commands/project.py

```diff
@@ -212,38 +212,142 @@ def __init__(self):
             _wrap('''
             Updates all projects according to the manifest file, `west.yml`,
             in the manifest repository.
-            All projects will be checked out in detached HEAD and the local
-            manifest-rev branch will be reset hard to same HEAD.
 
-            Any local branches will not be altered by this command.
+            By default:
 
-            This command will not touch the manifest repository.
+            1. West itself is updated to the newest revision in the manifest
```

Comment:
maybe: `to the revision in the manifest file. If revision is a branch, west will update to the tip of the branch`

cause, the revision in the manifest may not be newest revision in the west repo.

Reply:
Great suggestion

Reply:
done

Reply:
Is this done? I still see: `1. West itself is updated to the newest revision in the manifest file.`

Reply:
botched force push, now fixed

Reply:
:+1: